users/api/serializers.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework.serializers import ModelSerializer
+
+from users.models import Profile
+
+
+class ProfileDetailSerializer(ModelSerializer):
+    class Meta:
+        model = Profile
+        fields = ['user', 'image', 'bio', 'year', 'branch']
+
+
+class ProfileUpdateSerializer(ModelSerializer):
+    class Meta:
+        model = Profile
+        fields = ['image', 'bio', 'year', 'branch']
+
+
+class ProfileListSerializer(ModelSerializer):
+    class Meta:
+        model = Profile
+        fields = ['user', 'image', 'bio', 'year', 'branch']
```

Comment:
Why is the ListSerializer required? Is there any place where we would require a list of all profiles?

Reply:
Nope, I don't think so, will get it removed.